microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -283,18 +279,8 @@ Whether to include event initiators in the Timeline.
 <!-- todo: need text -->
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 108.
-*  This checkbox is present in Microsoft Edge Stable 106.
-
-
-<!-- ====================================================================== -->
-## Timeline: WebGL-based flamechart
```

Comment:
Policy:  Should we keep this entry, for users of older releases of Stable?  It would say:

*  This checkbox is not present in Microsoft Edge Canary 109.
*  This checkbox is not present in Microsoft Edge Stable 107.

Do some users still need such older entries?

Reply:
Plan of Record: delete entries when they don't appear in any of the latest channels.